communication.rst

```diff
@@ -6,6 +6,19 @@ Following Python's Development
 Python's development is communicated through a myriad of ways, mostly through
 mailing lists, but also other forms.
 
+
+Standards of behaviour in these communication channels
+------------------------------------------------------
+We try to foster environments of mutual respect, tolerance and encouragement,
+as described in the PSF's `Diversity Statement`_. Abiding by the guidelines
+in this document and asking questions or posting suggestions in the
+appropriate channels are an excellent way to get started on the mutual respect
+part, greatly increasing the chances of receiving tolerance and encouragement
+in return.
+
+.. _Diversity Statement: https://www.python.org/psf/diversity/
```

Comment:
Note: this section isn't new, I just pulled it up from where it was getting buried a bit with the specific sections on the mailings *and* Discourse *and* Discord all coming first and containing quite a bit of text.